data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -820,6 +827,28 @@
     "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
     "components": [ [ [ "salt", 210 ] ], [ [ "chem_sulphuric_acid", 1 ] ], [ [ "water_clean", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "chem_muriatic_acid",
+    "id_suffix": "by_electrolysis",
+    "byproducts": [ [ "lye", 2 ] ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "cooking",
+    "skills_required": [ "electronics", 2 ],
+    "difficulty": 5,
```

Comment:
I think igniting chlorine and hydrogen successfully with an electric arc welder in a chemical process probably requires a bit more than cooking 5.

Reply:
Indeed, I suppose one also needs to know how welder works and how to use it.